src/validation_provider/index.ts

```diff
@@ -20,28 +21,31 @@ export class ValidationProvider implements Disposable {
           if (!e.affectsConfiguration("dbt")) {
             return;
           }
-          this.validateCredentials();
+          this.validateCredentials(false);
```

Comment:
it will work differently from before as it was showing an error message. Is that intentional?

Reply:
not sure what you mean. i am still getting error notification if credentials are wrong.